### DISCUSSION TITLE
Testing harnesses manager with composition API (Quasar v2)

### DISCUSSION THREAD

**Discussion by doic** [quasarframework/quasar#9610] [Quasar App Extensions]
Hello,

I'm on Quasar v2 with Composition API, I installed the testing manager (quasar ext add @quasar/testing) but apparently, everything is set to run with Option API, am I wrong ?
Is there a way to reinstall the testing manager for Composition API ?

**Comment by ontwikkelfabriek** [comment]
@doic 

I think you need the `next` branch for Qv2:
https://github.com/quasarframework/quasar-testing/tree/next

But to be sure I would ask those kind of questions at the `quasar-testing` github instead of here.
